deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -507,20 +511,23 @@ def build_graph(self):
     batch_norm2 = BatchNorm(in_layers=[gc2])
     gp2 = GraphPool(in_layers=[batch_norm2, self.degree_slice, self.membership]
                     + self.deg_adjs)
-    dense = Dense(out_channels=128, activation_fn=None, in_layers=[gp2])
+    dense = Dense(out_channels=128, activation_fn=tf.nn.relu, in_layers=[gp2])
     batch_norm3 = BatchNorm(in_layers=[dense])
-    gg1 = GraphGather(
+    readout = GraphGather(
         batch_size=self.batch_size,
         activation_fn=tf.nn.tanh,
         in_layers=[batch_norm3, self.degree_slice, self.membership] +
         self.deg_adjs)
 
+    if self.error_bars == True:
```

Comment:
```self.error_bars``` is set on 482 I believe? Is there a better way to do that?

```dropout_prob``` is fixed because inverse model precision is a function of it. Right now that term is pre-computed and is validated in this configuration.